card-gateway/Gateway/Request/Builder/CardPaymentBuilder.php

```diff
@@ -31,8 +46,12 @@ public function build(array $buildSubject)
         return [
             RequestFields::CARD => $payment->getData(CardPaymentInterface::NUMBER),
             RequestFields::CARD_CVV => $payment->getData(CardPaymentInterface::CVV),
-            RequestFields::CARD_EXP_MONTH => $payment->getData(CardPaymentInterface::EXPIRATION_MONTH),
-            RequestFields::CARD_EXP_YEAR => $payment->getData(CardPaymentInterface::EXPIRATION_YEAR),
+            RequestFields::CARD_EXP_MONTH => $this->cardPaymentDataFormatter->format(
```

Comment:
Better to use method name to have formatter for a year and separately for a month.

Reply:
@mcspronko something like `getFormattedYear($value)` and `getFormattedMonth($value`) ?